File: examples/04_sound/script/main.lua
local texture = Texture()
local font = Font()
font:Load(16)
font:RenderText(texture, "Press SPACE key to play sound", Color(1, 1, 1, 1))
local draw2d = Draw2D(texture)
draw2d.scale = texture:Size()
local sound = Sound()
sound:Load("shot.wav")



-- Set scale to texture size

function Update()
    if Keyboard.IsPressed(Keyboard.SPACE) then
        sound:Play()
    end
end

function Draw()
    -- Draw texture
    draw2d:Draw()
end

File: examples/04_sound/script/main.py
from sinen import *
texture = Texture()
font = Font()
draw2d = Draw2D()
hello_sound = Sound()
hello_sound.load("shot.wav")

# Create a texture
font.load(16)
font.render_text(texture, "Press SPACE key to play sound", Color(1, 1, 1, 1))

# Create a draw2D
draw2d = Draw2D(texture)

# Set scale to texture size
draw2d.scale = texture.size()

def update():
  if Keyboard.is_pressed(Keyboard.SPACE):
    hello_sound.play()

def draw():
  # Draw texture
  draw2d.draw()
